languages/elixir/exercises/concept/basics/lib/lasagna.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Lasagna do
+  # TODO: define the 'expected_minutes_in_oven' function
```

Comment:
If unsure if it would make sense to include the arity here too, to pair it with the name specified in the task.

```suggestion
  # TODO: define the 'expected_minutes_in_oven/0' function
```

Then again, this might just be confusing people.